lib/oeqa/runtime/sanity/baseos.py

```diff
@@ -102,7 +102,33 @@ def test_baseos_systemd_process(self):
             status=1
         self.assertEqual(status, 0, msg="Error messages: %s" % output)
 
+    def test_baseos_systemd_boot_error(self):
+        ''' check systemd boot journal error''' 
+        known_issues_list = [
+                          "GPT: Use GNU Parted to correct GPT errors",
+                          "kernel",
```

Comment:
I find "kernel" a bit too generic. It'll exclude any error in the log, whether it is from the kernel or about the kernel.

What you want to exclude are lines like:
```
Apr 28 00:41:28 edison kernel: some error
```
Better use regular expressions instead of sub-string searches.